src/main/avro/uk/gov/homeoffice/digital/sas/timecard/time_card.avsc

```diff
@@ -33,15 +33,15 @@
             "name": "time_entries", 
```

Comment:
No specific comment on these changes but I do want to catch up on the inclusion on avro

Reply:
I'm going to remove avro